src/lib.rs

```diff
@@ -1,14 +1,179 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+pub use tracing;
+
+#[cfg(not(tokio_unstable))]
+mod tower_actor {
+    compile_error!("Tower actor must be compiled with a `--cfg tokio_unstable` rustc flag");
+}
+
+pub use tower_actor::*;
+
+#[cfg(tokio_unstable)]
+mod tower_actor {
+    pub(crate) mod actor;
+    pub(crate) mod future;
+    pub(crate) mod message;
+
+    pub use actor::*;
+    pub use future::*;
+    pub use message::*;
```

Comment:
The only tokio_unstable feature used here is the named task used to spawn the actor, right?

I think it'd be cleaner to make the crate compile unconditionally, and limit the cfg-gating to the spawn call, using the cfg to switch between either calling spawn directly and ignoring the task name, or using the unstable Builder API if it's available. (Maybe we'll get an unused variable warning in the unavailable case, I think that doesn't matter too much)